### PR TITLE
AC Test Fix - LRAC-10809 and LRAC-10805 | master

### PR DESCRIPTION
**Pull request by ViniLopes20**
https://issues.liferay.com/browse/LRAC-10809
https://issues.liferay.com/browse/LRAC-10805
[Testray Result 1](https://testray.liferay.com/home/-/testray/case_results/index?p_p_state=normal&orderByCol=status_sortable&orderByType=asc&testrayBuildId=1650295165)
[Testray Result 2](https://testray.liferay.com/home/-/testray/case_results/index?p_p_state=normal&orderByCol=status_sortable&orderByType=asc&testrayBuildId=1651276834)
[Testray Result 3](https://testray.liferay.com/home/-/testray/case_results/index?p_p_state=normal&orderByCol=status_sortable&orderByType=asc&testrayBuildId=1652352147)

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:sf

**Review by joshitagaki — APPROVED** [review]
LGTM!

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsForms.testcase [hunk 1]
```diff
@@ -71,17 +75,13 @@ definition {
...
-
 			var portalURL = PropsUtil.get("portal.url");
-
-			var siteName = TestCase.getSiteName(siteName = "${siteName}");
+			var dataSourceName = StringUtil.extractFirst("${assignedPropertyName}", " ");
 
 			ACAssets.viewAssetAppearsOnMetric(
 				canonicalURL = "${portalURL}/web/site-name",
 				cardTitle = "Asset Appears On",
-				pageNmae = "AC Page - Site Name - ${siteName}");
```
Comment (christianstokes) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsForms.testcase:84]: pageNmae should be pageName

**Review by christianstokes — CHANGES_REQUESTED** [review]
Hey @ViniLopes20 the code looks great!

It looks like there is a mistaken spelling "pageNmae" that affects a few different tests (not just this one). 😄 It should be "pageName" instead.

Do you mind adding a commit to fix all the instances of pageNmae? Then I think it can be submitted, it looks great. 👏 

pageNmae

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsForms.testcase (hunk 1 above)
Comment (ViniLopes20) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsForms.testcase:84]: Ohhh ok, no problem. I will make this change, thanks for noticing this 😄

**Comment by ViniLopes20** [comment]
ci:forward

**Comment by ViniLopes20** [comment]
Hi team, Thanks for reviewing!

I'm forwarding it!

**Pull request closed without merging**